python/hidet/utils/model_translator/utils.py

```diff
@@ -0,0 +1,31 @@
+# %%
```

Comment:
We also have the color support at https://github.com/hidet-org/hidet/blob/a1bd18cac4b959d63a8bc7c371e174d3cabe81ca/python/hidet/utils/py.py#L61-L121

We can consider reusing it.